scripts/run_e2e_tests.py

```diff
@@ -464,7 +467,9 @@ def main(args=None):
     update_community_dashboard_status_in_feconf_file(
         FECONF_FILE_PATH, parsed_args.community_dashboard_enabled)
 
-    if not parsed_args.skip_build:
+    if parsed_args.skip_build:
+        build.modify_constants(prod_env=parsed_args.prod_env)
```

Comment:
Do we need `prod_env`

Reply:
Not really, but I have modified `build.modify_constants` the way that the keywords are always forced in order to force the devs to use the keyword when `True` or `False` is passed.